app_callbacks.py

```diff
@@ -187,7 +187,7 @@ def run_queries(repos):
         # jobs are either all ready
         if all([j.successful() for j in jobs]):
             logging.info([j.status for j in jobs])
-            return "Data Ready", "success"
+            return "Data Ready", "#B5B682"
```

Comment:
understood, also should have an in-line comment of what this color looks like for debugging / readability.